weblate/utils/requirements.py

```diff
@@ -169,10 +172,66 @@ def get_versions():
     return result
 
 
+def get_db_version():
+    if connection.vendor.upper() == "POSTGRESQL":
+        try:
+            cursor = connection.cursor()
+            cursor.execute("SHOW server_version")
+            version = cursor.fetchall()[0]
+            version = version[0].split(" ")[0]
+            result = (
+                connection.vendor,
+                "https://www.postgresql.org/",
+                version,
+            )
+            cursor.close()
+        except RuntimeError:
+            raise ImproperlyConfigured(
+                "Failed to get a database version. please install a database."
+            )
+    else:
+        try:
+            cursor = connection.cursor()
+            result = (
+                connection.vendor,
+                "https://www.mysql.com/",
+                cursor.connection.get_server_info().split("-", 1)[0],
+            )
+            cursor.close()
+        except RuntimeError:
+            raise ImproperlyConfigured(
+                "Failed to get a database version. please install a database."
+            )
+
+    return result
+
+
+def get_cache_version():
+    try:
+        if (
+            settings.CACHES.get("default", {}).get("BACKEND")
+            == "django_redis.cache.RedisCache"
+        ):
+            version = cache.client.get_client().info()["redis_version"]
+            result = ("Redis", "https://redis.io/", version)
+        else:
+            raise ImproperlyConfigured("please install a redis server.")
```

Comment:
I'm not sure about this exception. Weblate will probably work just fine with other cache backend, so raising an exception seems too strict here.

Reply:
I will remove this exception. But I'm not sure, how to get the cache version/name for some other backend.There can be many such services. OR  I have to show only cache version if the backend is `django_redis.cache.RedisCache`

Reply:
There is no reason to cover all options, getting it only for redis is good enough as that will cover most use cases. But it should not break in others.

Reply:
@nijel - I have removed the `ImproperlyConfigured` exception. Also, I tested the code with another cache namely Memcached and its working fine.